### DISCUSSION TITLE
Problem with message_handler

### DISCUSSION THREAD

**Discussion by Vamashi** [eternnoir/pyTelegramBotAPI#1741] [General]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?  4.7.0 (latest)

2. What OS are you using? Windows

3. What version of python are you using? 22.2.2

I tried alot of codes, no one working, i tried with code from documentation, it also not working :(
I think problem with @message_handler, because bot starting polling, can send messages, but if i send a message to him, he not response (no print, no messages)

`import telebot

bot = telebot.TeleBot("token")

@bot.message_handler(commands=['start', 'help'])
def send_welcome(message):
	bot.reply_to(message, "Howdy, how are you doing?")

@bot.message_handler(func=lambda message: True)
def echo_all(message):
	bot.reply_to(message, message.text)

bot.infinity_polling()`

**Comment by Vamashi** [comment]
It not working with any bot

**Comment by Badiboy** [comment]
It definitely working with any bot.
All code in examples are fully functional.

**Comment by Vamashi** [comment]
> It definitely working with any bot. All code in examples are fully functional.

But why is no one bot working for me? They worked yesterday, but today it's not! No one bot

**Comment by Badiboy** [comment]
You want us, mainteiners of the library, to say you what happened in your surronding yesterday? Seriously?

**Comment by coder2020official** [comment]
Not a lib bug. The lib does what it is supposed to do. Try getting updates manually;

**Comment by coder2020official** [comment]
Let's wait for his response

**Comment by coder2020official** [comment]
We can move it to discussions

**Comment by coder2020official** [comment]
@Vamashi ?

**Reply by Vamashi** [reply]
Sorry, i fixed it, problem was with bot
